lib/node_modules/@stdlib/math/base/special/logit/test/test.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/utils/is-nan' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var logit = require( './../lib' );
+
+
+// FIXTURES //
+
+var small = require( './fixtures/python/small.json' );
+var medium = require( './fixtures/python/medium.json' );
+var large = require( './fixtures/python/large.json' );
+
+
+// FUNCTIONS //
+
+function almostEqual( a, b, tol ) {
+	var delta = abs( a - b );
+	tol = tol * Math.max( 1.0, abs( a ), abs( b ) );
+	return ( delta <= tol );
+}
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof logit, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` when provided `NaN`', function test( t ) {
+	var y = logit( NaN );
+	t.equal( isnan( y ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `NaN` when provided a number outside `[0,1]`', function test( t ) {
+	var y = logit( 1.2 );
+	t.equal( isnan( y ), true, 'returns NaN' );
+	y = logit( -0.1 );
+	t.equal( isnan( y ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `-Infinity` when provided `0`', function test( t ) {
+	var y = logit( 0.0 );
+	t.equal( y, NINF, 'returns -Infinity' );
+	t.end();
+});
+
+tape( 'the function returns `+Infinity` when provided `1`', function test( t ) {
+	var y = logit( 1.0 );
+	t.equal( y, PINF, 'returns +Infinity' );
+	t.end();
+});
+
+tape( 'the function evaluates the logit of `x` for the interval `(0,0.25]`', function test( t ) {
+	var expected;
+	var bool;
+	var tol;
+	var x;
+	var y;
+	var i;
+
+	tol = EPS;
+
+	expected = small.expected;
+	x = small.x;
+	for ( i = 0; i < x.length; i++ ) {
+		y = logit( x[i] );
+		if ( y === expected[i] ) {
+			t.equal( y, expected[i], 'x: '+x[i]+', y: '+y+', expected: '+expected[i] );
+		}  else {
+			bool = almostEqual( y, expected[i], tol );
+			t.ok( bool, 'within tolerance. x: '+x[i]+'. y: '+y+'. Expected: '+expected[i]+'. Tolerance: '+tol+'.' );
```

Comment:
This is not the actual tolerance. The actual tolerance is scaled. Another reason we should not be using a functional definition for "almost equal".

 Should also include the `delta`.